enterprise/backend/src/metabase_enterprise/serialization/api.clj

```diff
@@ -140,7 +152,22 @@
                             :dirname                  dirname}
         {:keys [archive
                 log-file
+                report
+                error-message
                 callback]} (serialize&pack opts)]
+    (snowplow/track-event! ::snowplow/serialization-export api/*current-user-id*
+                           {:source          "api"
+                            :duration        (long (/ (- (System/currentTimeMillis) start) 1000))
+                            :count           (count (:seen report))
+                            :collection      (str/join "," (map str collection))
```

Comment:
Nitpick: you do not need to map over `str`, it is implicit in the `join`.

Reply:
ohh indeed! nice!